### DISCUSSION TITLE
Problem solved. An error occurred when attempting to deploy SimpleStorage.sol using keystore: Error: No such file or directory (os error 2)

### DISCUSSION THREAD

**Discussion by Mud-Yun** [Cyfrin/foundry-full-course-cu#2360] [Q&A]
The command should be :
forge script script/DeploySimpleStorage.s.sol:DeploySimpleStorage --rpc-url http://localhost:8545 --keystore defaultKey --broadcast -vvvv

not :
forge script script/DeploySimpleStorage.s.sol:DeployFundMe --rpc-url http://localhost:8545 --keystore defaultKey --broadcast -vvvv

**Comment by EngrPips** [comment]
Hey @Mud-Yun, Thanks for this contribution.
